lib/traces/backend/open_telemetry/interface.rb

```diff
@@ -66,13 +66,18 @@ def trace_context=(context)
 				end
 				
 				def trace_context(span = ::OpenTelemetry::Trace.current_span)
+					# state = baggage.values(context: span.context)
 					if span_context = span.context
-						state = baggage.values(context: span.context)
```

Comment:
`state` was an unused variable and the baggage api has changed, if this is important I need to understand the otel baggage concept and implement.

Reply:
I don't think it's important. The goal is to create a context which can be sent over the wire, e.g. over an HTTP request or encoded into a database query comment etc.